doc/Makefile

```diff
@@ -57,5 +57,5 @@ clean:
 	rm -rf api/generated
 	rm -rf gallery
 	rm -rf tutorials
-	rm -rf get_started
+	rm -rf intro
```

Comment:
Also need to update `.gitignore`.

Reply:
Updated in commit https://github.com/GenericMappingTools/pygmt/pull/3571/commits/d859020df5ddfa16312973a5c1330d443f5cd13e.